CONTRIBUTING.md

```diff
@@ -84,6 +84,25 @@ export TESTCONTAINERS_RYUK_PRIVILEGED=true
 
 when running TestContainer-based Model Registry Python tests (for more information, see [here](https://pypi.org/project/testcontainers/#:~:text=TESTCONTAINERS_RYUK_PRIVILEGED)).
 
+#### Fedora
+
+Fedora requires further setup to make testcontainers work with Podman, the following steps are required:
+
+- start the podman socket service
+
+```sh
+systemctl --user start podman.socket
+```
+
+- set the environment variables
+
+```sh
+export DOCKER_HOST=unix://$XDG_RUNTIME_DIR/podman/podman.sock
+export TESTCONTAINERS_RYUK_PRIVILEGED=true
```

Comment:
@Al-Pragliola can you kindly double-check if it's about `TESTCONTAINERS_RYUK_PRIVILEGED` or `TESTCONTAINERS_RYUK_DISABLED`, please?

see also: https://github.com/kubeflow/model-registry/pull/269#issuecomment-2312431157

Reply:
I removed the export to privileged, I may have had an unclean state when I tested the feature, it works now just with the change I made in the code and the export to DOCKER_HOST.

Reply:
btw it works with either of them set to true or false